### PR TITLE
feat(cocktailprev): we reorganised our components

### PR DESCRIPTION
**Pull request by lulup52**
we create a new component to manage our result list

### CODE REVIEW

File: src/components/CocktailList.jsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import Cocktail from './Cocktail';
-
+import CocktailPrev from './CocktailPrev';
```
Comment (rocambille) [inline src/components/CocktailList.jsx]: Prev: Previous ? Preview ? please use full words :)

File: src/App.jsx [hunk 2]
```diff
@@ -10,6 +10,8 @@ import CocktailResult from './components/CocktailResult';
 import CocktailDescription from './components/CocktailDescription';
 
 
+
```
Comment (rocambille) [inline src/App.jsx]: remove multiple empty lines

**Review by rocambille — CHANGES_REQUESTED** [review]

File: src/components/CocktailList.jsx (hunk 1 above)
Comment (lulup52) [reply src/components/CocktailList.jsx]: done

File: src/App.jsx (hunk 2 above)
Comment (lulup52) [reply src/App.jsx]: done

**Pull request merged**